app/services/dual-output/dual-output.ts

```diff
@@ -449,20 +429,24 @@ export class DualOutputService extends PersistentStatefulService<IDualOutputServ
   }
 
   convertSingleOutputToDualOutputCollection() {
+    this.SET_IS_LOADING(true);
     // establish vertical context if it doesn't exist
-    if (this.state.dualOutputMode && !this.videoSettingsService.contexts.vertical) {
+    if (!this.videoSettingsService.contexts.vertical) {
       this.videoSettingsService.establishVideoContext('vertical');
     }
 
-    // convert all scenes in the single output collection to dual output
-    this.scenesService.views.scenes.forEach(scene => {
-      this.createPartnerNodes(scene.id);
-    });
+    try {
+      // convert all scenes in the single output collection to dual output
+      this.scenesService.views.scenes.forEach(scene => {
+        this.createPartnerNodes(scene.id);
+      });
+    } catch (error: unknown) {
+      console.error('Error converting to single output collection to dual output: ', error);
```

Comment:
it feels like there should maybe be some sort of cleanup here to safely return the scene collection to a state of single output, but maybe that out of scope for this PR